test/integration/patients/patient/archive.js

```diff
@@ -418,46 +470,69 @@ context('patient archive page', function() {
   });
 
   specify('work with work:team:manage permission', function() {
+    const currentClinican = getCurrentClinician({
+      id: '11111',
+      relationships: {
+        role: getRelationship(roleTeamEmployee),
+        team: getRelationship(teamCoordinator),
+      },
+    });
+
     cy
       .routesForPatientAction()
       .routeCurrentClinician(fx => {
-        fx.data.relationships.role = { data: { id: '77777' } };
-        fx.data.relationships.team = { data: { id: '11111', type: 'teams' } };
+        fx.data = currentClinican;
 
         return fx;
       })
       .routeWorkspaceClinicians(fx => {
-        fx.data = _.first(fx.data, 2);
+        const nonTeamMemberClinician = {
```

Comment:
seems like this should be in a `getClinician`?  Also we could not set the id and then use `nonTeamMemberClinician.id` in the below tests

Reply:
I updated this to `getClinician()`. Good call, I missed that.

Looks like we're required to include an `id` with `getClinician()`: https://github.com/RoundingWell/care-ops-frontend/blob/develop/test/support/api/clinicians.js#L31. I don't fully understand why though.